.travis.yml

```diff
@@ -1,16 +1,27 @@
-language: python
+language: php
 
-python:
-    - "2.7"
+php: 5.6
 
 sudo: false
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+    - _build
 
-script: sphinx-build -nW -b html -d _build/doctrees . _build/html
+before_install:
+  - python --version
+  - pip install -U setuptools
```

Comment:
there is a problem with the pip install command (see first commit travis build). This was a try to fix it, which didn't work out very well...